App.tsx

```diff
@@ -1,405 +1,49 @@
-
-// components
-import { headerOptions } from "@components/headerComponents/headerOptions"
-
-// navigation
-import { createBottomTabNavigator } from "@react-navigation/bottom-tabs"
-import {
-  createDrawerNavigator,
-  DrawerNavigationOptions
-} from "@react-navigation/drawer"
 import { NavigationContainer } from "@react-navigation/native"
 import { createStackNavigator } from "@react-navigation/stack"
-import ChallengeStack from "@screens/adminScreens/ChallengeStack"
-import ReportScreen from "@screens/adminScreens/ReportScreen"
-import VerificationRequestsScreen from "@screens/adminScreens/VerificationRequestsScreen"
-import ComplianceScreen from "@screens/ComplianceScreen"
-import ConversationScreen from "@screens/ConversationScreen"
-import CustomSidebarMenu from "@screens/CustomSidebarMenu"
-import FriendScreen from "@screens/FriendScreen"
-import ImageScreen from "@screens/ImageScreen"
-import MessageScreen from "@screens/MessageScreen"
-import ProfileScreen from "@screens/ProfileScreen"
-import VerificationScreen from "@screens/VerificationScreen"
-import MainStack from "@stacks/MainStack"
-import SettingsStack from "@stacks/SettingsStack"
-
-// react
-import AsyncStorage from "@react-native-async-storage/async-storage"
-import { Audio } from "expo-av"
-import * as Notifications from "expo-notifications"
-import { StatusBar } from "expo-status-bar"
-import React, { useEffect, useRef, useState } from "react"
-import {
-  ActivityIndicator,
-  Alert,
-  AppState,
-  AppStateStatus,
-  Platform,
-  StyleSheet,
-  UIManager,
-  useWindowDimensions,
-  View
-} from "react-native"
-
-// sign in
-import { AppQueryClientProvider } from "@components/AppQueryClientProvider"
-import ConfirmSignIn from "@components/loginComponents/ConfirmSignIn"
-import ConfirmSignUp from "@components/loginComponents/ConfirmSignUp"
-import ForgotPassword from "@components/loginComponents/ForgotPassword"
-import Greetings from "@components/loginComponents/Greetings"
-import RequireNewPassword from "@components/loginComponents/RequireNewPassword"
-import SignIn from "@components/loginComponents/SignIn"
-import SignUp from "@components/loginComponents/SignUp"
-import VerifyContact from "@components/loginComponents/VerifyContact"
+import { TiFQueryClientProvider } from "@components/TiFQueryClientProvider"
 import { useAppFonts } from "@hooks/Fonts"
-import { makeLinkingConfig } from "@lib/linkingConfig"
-import { ExpoUserNotifications } from "@lib/UserNotifications"
-import ActivitiesStack, { TabNavigation } from "@stacks/ActivitiesStack"
-import { SetDependencyValue } from "./lib/dependencies"
-import { userIdDependencyKey } from "./lib/MiscDependencyKeys"
+import { TabNavigation } from "@stacks/ActivitiesStack"
 import { SafeAreaProvider } from "react-native-safe-area-context"
-
-if (
-  Platform.OS === "android" &&
-  UIManager.setLayoutAnimationEnabledExperimental
-) {
-  UIManager.setLayoutAnimationEnabledExperimental(true)
```

Comment:
Need this to allow for layout animations on android https://reactnative.dev/docs/layoutanimation